### DISCUSSION TITLE
PaddleOCR/ppocr/blog/ocr_book

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13572] [Q&A]
# PaddleOCR/ppocr/blog/ocr_book

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/ppocr/blog/ocr_book.html

**Comment by whizhuii** [comment]
非常抱歉，我非常想加群，但是我花了一些时间没有找到首页的二维码。如果是我没找到，希望能把指引做得更清楚一点。

**Reply by GreatV** [reply]
@ZcyhFromC paddleocr现在没有群，可以加一下paddlex的群。中文电子书链接目前还没有，只有英文版本的链接。后续可能会提供。

**Reply by GreatV** [reply]
@ZcyhFromC 已更新

**Reply by whizhuii** [reply]
非常感谢

**Comment by sssssshf** [comment]
有没有分离的教程,比如我只希望获取ocr  文字识别的推理部分.

**Reply by GreatV** [reply]
目前没有
